tests/test_redirects.py

```diff
@@ -206,58 +122,33 @@ def test_redirect_for_firefox_aliases(self, base_url, product_alias):
             url_scheme = 'http'
             if product_alias['product_name'] == 'firefox-beta-stub':
                 url_scheme = 'https'
-            Assert.equal(
-                response.status_code,
-                requests.codes.ok,
-                'Redirect failed with HTTP status %s. \n %s' %
-                (response.status_code, self.response_info_failure_message(
-                    base_url,
-                    param,
-                    response))
-            )
-            Assert.equal(
-                parsed_url.scheme,
-                url_scheme,
-                'Failed, redirected to incorrect scheme %s. \n %s' %
-                (parsed_url.scheme, self.response_info_failure_message(
-                    base_url,
-                    param,
-                    response))
-            )
-            Assert.true(
-                parsed_url.netloc in ['download.cdn.mozilla.net', 'edgecastcdn.net',
-                        'download-installer.cdn.mozilla.net', 'cloudfront.net',
-                        'ftp.mozilla.org']
-               ,
-                'Failed, redirected to unknown host %s. \n %s' %
-                (parsed_url.netloc, self.response_info_failure_message(
-                    base_url,
-                    param,
-                    response))
-            )
-            Assert.not_equal(
-                self.get_x_backend_server(response),
-                'Unknown',
-                'Failed, x-backend-server was not in the response object %s' %
-                (self.response_info_failure_message(base_url,
-                    param,
-                    response))
-            )
+            assert (requests.codes.ok == response.status_code,
+                    'Redirect failed with HTTP status %s. \n %s' %
+                    (response.status_code, self.response_info_failure_message(base_url, param, response)))
+
+            assert (url_scheme == parsed_url.scheme,
+                    'Failed to redirect to the correct scheme. %s' %
+                    (self.response_info_failure_message(base_url, param, response)))
+
+            assert (parsed_url.netloc in ['download.cdn.mozilla.net', 'edgecastcdn.net',
+                    'download-installer.cdn.mozilla.net', 'cloudfront.net', 'ftp.mozilla.org'],
+                    'Failed, redirected to unknown host %s. \n %s' %
+                    (parsed_url.netloc, self.response_info_failure_message(base_url, param, response)))
```

Comment:
Nit: No need to put the actual value into the failure message as pytest will do this for us.